From f637af3a7c24fc94709c58ead6991c615a59da78 Mon Sep 17 00:00:00 2001
From: Jack Tan <jack.tan@windriver.com>
Date: Fri, 14 Feb 2014 17:29:42 +0800
Subject: [PATCH 27/58] bcm-xlp2: add a new directory 'lib'

It's to store the core libraries from SDK_DIR/libraries

Signed-off-by: Jack Tan <jack.tan@windriver.com>
---
 arch/mips/netlogic/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/netlogic/Makefile b/arch/mips/netlogic/Makefile
index 69def59..006024d 100644
--- a/arch/mips/netlogic/Makefile
+++ b/arch/mips/netlogic/Makefile
@@ -3,3 +3,4 @@ obj-$(CONFIG_CPU_XLR)		+=	xlr/
 obj-$(CONFIG_CPU_XLP)		+=	xlp/
 obj-$(CONFIG_CPU_XLP)		+=	dts/
 obj-$(CONFIG_KVM)		+=	kvm/
+obj-$(CONFIG_NLM_XLP_BOARD)	+=	lib/
-- 
1.7.9.5

